bedmaker.py

```diff
@@ -19,7 +21,8 @@
 parser.add_argument("-r", "--rfg-config", help="file path to the genome config file", type=str)
 parser.add_argument("-o", "--output-file", help="path to the output BED files", type=str)
 parser.add_argument("-s", "--sample-name", help="name of the sample used to systematically build the output name", type=str)
-
+parser.add_argument("--output-bigbed", help="path to the output bigBED files", type=str)
```

Comment:
Sometimes you used BigBed and sometimes BigBED. It seems UCSC referse to it as `bigBed` rather than `bigBED` so let's just follow that standard... so that's BED and bigBed, I guess:

- https://genome.ucsc.edu/goldenPath/help/bigBed.html 
- https://genome.ucsc.edu/goldenPath/help/bigBed.html